data/data/openstack/variables-openstack.tf

```diff
@@ -0,0 +1,565 @@
+variable "tectonic_autoscaling_group_extra_tags" {
```

Comment:
We are moving away from `tectonic_` prefixes now that we're the OpenShift installer.  Can you drop the prefix?  On a quick skim, the names look specific-enough without them.

And actually, this specific variable seems to be unused.  Can you drop it altogether?